src/App.vue

```diff
@@ -664,19 +668,62 @@ export default Vue.extend({
         autoSaveStateToWebLocalStorage(reason: SaveRequestReason) : void {
             // save the project to the localStorage (WebStorage)
             if (!this.appStore.debugging && typeof(Storage) !== "undefined") {
-                localStorage.setItem(this.localStorageAutosaveKey, this.appStore.generateStateJSONStrWithCheckpoint(true));
-                // If that's the only element of the auto save functions, then we can notify we're done when we save for loading
-                if(reason==SaveRequestReason.loadProject && projectSaveFunctionsState.length == 1){
-                    this.$root.$emit(CustomEventTypes.saveStrypeProjectDoneForLoad);
+                if(reason == SaveRequestReason.saveSettings){
+                    // Save the settings
+                    localStorage.setItem(AutoSaveKeyNames.settingsState, JSON.stringify(this.settingsStore.$state));
+                }
+                else{
+                    localStorage.setItem(this.localStorageAutosaveEditorKey, this.appStore.generateStateJSONStrWithCheckpoint(true));
+                    // If that's the only element of the auto save functions, then we can notify we're done when we save for loading
+                    if(reason==SaveRequestReason.loadProject && projectSaveFunctionsState.length == 1){
+                        this.$root.$emit(CustomEventTypes.saveStrypeProjectDoneForLoad);
+                    }
                 }
             }
         },
 
+        setStrypeLocale() {
+            // We need to retrieve Strype's language (session) if it exists in the localStorage.
+            // If we didn't retrieve it, we try to infer the browser's language and ask the user
+            // if they want to use the detected (supported) language. 
+            // If they refused or if we can't retrieve anything at all, we use English as default.
+            let strypeSessionLocale = "en"; // default locale
+            let canCheckBrowserLocale = false;
```

Comment:
I think "canCheck" is misleading here.  It's not whether we can but whether we should.  I'd go for "checkBrowserLocale" instead.